packages/admin-panel/src/editor/actions.js

```diff
@@ -47,7 +47,7 @@ export const openEditModal = ({ editEndpoint, fields }, recordId) => async (
     }
   } else {
     // set default values
-    for (const field of fields) {
+    fields.forEach(field => {
```

Comment:
In our updated style guide, it's very likely that we will in fact enable `for...of` loops! No need to change this back, of course